docs/generated/manifests/packages.json

```diff
@@ -645,40 +645,40 @@
     },
     "path": "/packages/esbuild"
   },
-  "eslint-plugin-nx": {
+  "eslint-plugin": {
     "githubRoot": "https://github.com/nrwl/nx/blob/master",
-    "name": "eslint-plugin-nx",
-    "packageName": "@nrwl/eslint-plugin-nx",
```

Comment:
This should stay the same until we publish it - `@nrwl/eslint-plugin-nx`

Reply:
Fixed :)